docs/docsite/rst/dev_guide/migrating_roles.rst

```diff
@@ -191,7 +191,7 @@ To migrate other role plugins to a collection:
 
 2. Update documentation to use the FQCN. Plugins that use ``doc_fragments`` need to use FQCN (for example, ``mydocfrag`` becomes ``mynamespace.mycollection.mydocfrag``).
 
-3. Update relative imports work in collections to start with a period.  For example, :file:`./filename` and :file:`../asdfu/filestuff` works but :file:`filename` in same directory must be updated to :file:`./filename`.
+3. Update relative imports work in collections to start with a period.  For example, :file:`./file name` and :file:`../asdfu/filestuff` works but :file:`file name` in same directory must be updated to :file:`./file name`.
```

Comment:
similarly remove these change as well, since it's representing an actual file name as 'filename'